lib/mindwendel_web/live/live_helpers.ex

```diff
@@ -12,6 +12,14 @@ defmodule MindwendelWeb.LiveHelpers do
     Enum.member?(brainstorming.moderating_users |> Enum.map(& &1.id), current_user.id)
   end
 
+  def has_ownership(idea, current_user) do
+    idea.user_id == current_user.id
+  end
+
+  def has_moderating_or_ownership_permission(brainstorming, idea, current_user) do
+    has_ownership(idea, current_user) or has_moderating_permission(brainstorming, current_user)
```

Comment:
similarly permissions also sounds like a perhaps fully different concept that should probably rather be part of the business logic vs. live view helpers

Reply:
where would you propose? context of a brainstorming also does not really sound right?

Reply:
I'd say `brainstorming` context is more appropriate, but it really depends on the application etc.... like you can say that the brainstorming or ideas context cares about ownership. You could make a new `ownership` context or since it's more general we could say that `permissions` is its own thing and make that into a context